README.md

```diff
@@ -215,6 +215,35 @@ Just like with the single parameter case you can just update the store and the U
 
 writing in the input will update the state and the URL at the same time.
 
+### Updating multiple parameters at once
+
+When you need to update multiple different parameters at once you will want to use the `update()` method to update the store to prevent multiple calls of `goto()`.
```

Comment:
This only happens when using `queryParameters` tho...can you please specify that in the documentation?

Reply:
Would it not happen if you had multiple queryParams? Doesn't it call goto the same way?